src/Tribe/Views/V2/Views/By_Day_View.php

```diff
@@ -486,15 +488,39 @@ protected function backfill_multiday_event_ids( array $events = [] ) {
 				continue;
 			}
 
+			$site_timezone     = Timezones::build_timezone_object();
+			$utc               = Timezones::build_timezone_object( 'UTC' );
+			$use_site_timezone = Timezones::is_mode( 'site' );
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 0ab56c44812c63b716f1e4a218ff719c</sub>